src/cli.ts

```diff
@@ -326,7 +326,7 @@ function logNetworkInfo(ctx: Context) {
   log(`assetId: ${ctx.avaxAssetID}`)
 }
 
-async function logValidatorInfo(ctx: Context) {
+export async function logValidatorInfo(ctx: Context) {

```

Comment:
since the function is export please add function description in comment as documentation.